File: delete_file.php
<?php
    session_start();    
    $tmpFile = $_SESSION["file"];
    session_write_close();
    
    $pyscript = "delete_file.py";
    $python = "C:\\Python27\\python.exe";
    exec($python." ".escapeshellarg($pyscript)." ".escapeshellarg($tmpFile));
    
    //unlink($file);
?>

File: delete_file.py
#!C:/Python27/python.exe
# -*- coding: utf-8 -*-

#-------------------------------------------------------------------------------
# Name:        module1
# Purpose:
#
# Author:      Liska_P
#
# Created:     09.10.2013
# Copyright:   (c) Liska_P 2013
# Licence:     <your licence>
#-------------------------------------------------------------------------------

import sys
import os

try:
    filename = sys.argv[1]
    os.remove(filename)
except:
    print "Error Delete_file"